packages/command-palette/src/use-command-palette.tsx

```diff
@@ -293,7 +318,18 @@ export const useCommandPalette = ( {
 
 		finalSortedCommands = finalSortedCommands.map( ( command: Command ) => {
 			const callback = ( params: CommandCallBackParams ) => {
-				const targetFunction = command?.siteFunctions?.onClick || command.callback;
+				let targetFunction;
+				const isMultiSiteCommand = command?.siteFunctions !== undefined;
+				if ( isMultiSiteCommand ) {
+					targetFunction = command?.siteFunctions?.onClick;
+					// We need to track the selected command event here because `command.siteFunctions.onClick`
+					// does not track anything (and it's not meant to).
+					trackSelectedCommand( command );
+				} else {
+					// We don't need to track the selected command event here because `command.callback`
+					// already does it (see how `finalSortedCommands` is created a few lines above).
+					targetFunction = command.callback;
```

Comment:
If the code reaches this point we should change the tracking from `isSelected` to `isCompleted`.  At line 301 we're always assigning  `isSelected` to the main callback.

Reply:
I think I solved this in https://github.com/Automattic/wp-calypso/pull/88285/commits/c91689a61ceb9638e378425472f8937737f26e8f